resources/windows/wix/project.wxs.erb

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="windows-1252"?>
+  <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+    <Product
+      Id="*"
+      UpgradeCode="<%= settings[:upgrade_code] %>"
+      Name="<%= settings[:product_name] %>"
+      Language="1033"
+      Codepage="1252"
+      Version="<%= @version.sub(/\.g[0-9a-z]{7}$/, '') %>"
```

Comment:
Maybe not in this PR but we should add a method that will return this value formatted already so we can pull the `sub` call out of this template.